gems/pending/appliance_console/database_configuration.rb

```diff
@@ -64,6 +64,9 @@ def activate
 
     def post_activation
       ServiceGroup.new.restart_services
+      evm_service = LinuxAdmin::Service.new("evmserverd")
+      evm_service.enable
+      evm_service.start
```

Comment:
this is good
Can we do:

```ruby
LinuxAdmin::Service.new("evmserverd").enable.start
```

Hmm, may require a change over in Service. I'll put PR in for that if it doesn't exist.

this requires test changes (`and_call_original`)